src/OrchardCore/OrchardCore.ResourceManagement.Abstractions/RequireSettings.cs

```diff
@@ -48,6 +50,7 @@ public bool HasAttributes
         /// <summary>
         /// The resource will be displayed in the head of the page
         /// </summary>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
all methods here are are quite small, it doesn't hurt to hint about inlining, especially when call site is Combine which calls all of them